astropy/time/core.py

```diff
@@ -464,6 +514,37 @@ def __repr__(self):
     def __str__(self):
         return str(getattr(self, self.format))
 
+    def strftime(self, format_spec):
+        """
+        Convert Time to a string or a numpy.array of strings according to a
+        format specification.
+        See `time.strftime` documentation for format specification.
+
+        Parameters
+        ----------
+        format_spec : string
+            Format definition of return string.
+
+        Returns
+        -------
+        formatted : string, numpy.array
+            String or numpy.array of strings formatted according to the given
+            format string.
+
+        """
+        formatted_strings = []
+        for sk in self.replicate('iso')._time.str_kwargs():
+            date_tuple = date(sk['year'], sk['mon'], sk['day']).timetuple()
+            datetime_tuple = (sk['year'], sk['mon'], sk['day'],
+                              sk['hour'], sk['min'], sk['sec'],
+                              date_tuple[6], date_tuple[7], -1)
+            formatted_strings.append(strftime(format_spec, datetime_tuple))
```

Comment:
Better to `import datetime` at the top and refer to this (and `strptime`) explicitly as `datetime.strftime`.